code/modules/clothing/glasses/engine_goggles.dm

```diff
@@ -85,6 +90,8 @@
 			show_shuttle()
 		if(MODE_PIPE_CONNECTABLE)
 			show_connections()
+		if(MODE_ATMOS_THERMAL)
+			atmos_thermal(user)
```

Comment:
would it be better to give these to the glasses atmos techs have shiftstart (t-ray I think?) to make engineering mesons less of a must-have?

Reply:
So just add the mode to existing goggles? That works for me